### PR TITLE
feat(Illustration): add Interview illustration

### PR DESCRIPTION
**Pull request by natalieweesh**
### Overview

add Interview illustration

Screenshot 2023-12-05 at 5 13 01 PM

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: [ABC-123]
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal     | [Portal PR](https://github.com/codecademy-engineering/mono/pull/4372)  | [Monolith Env](http://www.google.fr/ 'Named link title') |

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@55.10.7-alpha.8f43d3.0<br/>@codecademy/gamut-illustrations@0.41.2-alpha.8f43d3.0<br/>@codecademy/gamut-kit@0.6.371-alpha.8f43d3.0<br/>@codecademy/styleguide@66.8.7-alpha.8f43d3.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@55.10.7-alpha.c1c058.0<br/>@codecademy/gamut-illustrations@0.41.2-alpha.c1c058.0<br/>@codecademy/gamut-kit@0.6.371-alpha.c1c058.0<br/>@codecademy/styleguide@66.8.7-alpha.c1c058.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://657096f3080a4a07160f59ff--gamut-preview.netlify.app>https://657096f3080a4a07160f59ff--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/657096f3080a4a07160f59ff>Deploy Logs</a></p>

File: packages/gamut-illustrations/src/Interview.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import { IllustrationProps } from './types';
+
+export const Interview: React.FC<IllustrationProps> = ({
+  'aria-hidden': ariaHidden,
+  className,
+  height,
+  width,
+}) => (
+   <svg aria-hidden={ariaHidden}
```
[the remark this answers is not in the document]
Comment (natalieweesh) [reply packages/gamut-illustrations/src/Interview.tsx]: ooh thanks! updated 👍

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@55.10.7-alpha.6c9e7f.0<br/>@codecademy/gamut-illustrations@0.41.2-alpha.6c9e7f.0<br/>@codecademy/gamut-kit@0.6.371-alpha.6c9e7f.0<br/>@codecademy/styleguide@66.8.7-alpha.6c9e7f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6570bd17b8bd6f03182b88a7--gamut-preview.netlify.app>https://6570bd17b8bd6f03182b88a7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6570bd17b8bd6f03182b88a7>Deploy Logs</a></p>

**Pull request merged**